### DISCUSSION TITLE
S3 Upload

### DISCUSSION THREAD

**Discussion by bentovsky** [Budibase/budibase#5360] [Help]
Hi,

I'm trying to upload an image to S3. I've created a datasource that is working, but when I upload an image and save the content, no request is made to AWS S3. The data sent in the request is the image in base64 (not sure if this is correct).
I'm using the version 1.0.110.

Thank you

**Comment by melohagan** [comment]
Hi @bentovsky 

Are there any errors in the network tab when you upload / save? (Fn+F12 on Chrome)

**Reply by bentovsky** [reply]
No, I don't have any errors. When I upload an image, I can see the preview (I think this was solved a few days ago). When I save, I see the success message "Row saved", but no request is made to AWS S3.

**Comment by bentovsky** [comment]
I've realized that I wasn't defining an "upload file to s3" action. But now, when I save, I'm having the following error:
"Error uploading file: TypeError: e.getSignedDatasourceURL is not a function".

**Reply by melohagan** [reply]
We are going to investigate this further next week. 
In the meantime, would you be able to provide any additional error output from the network tab & console tab for that TypeError?

**Reply by bentovsky** [reply]
I'm not having any error on the console or network, the only error thrown is this one:
![image](https://user-images.githubusercontent.com/16797919/163809365-a87e826a-b367-49f7-90d7-9299dce37628.png)

**Reply by phil-lange** [reply]
Hello, we are facing the same issue, could you give a little status update?

**Reply by melohagan** [reply]
Hey @phil-lange We haven't figured this one out yet. Created a bug here: https://github.com/Budibase/budibase/issues/5876
